app/routes/interestgroups/components/InterestGroupDetail.js

```diff
@@ -36,12 +36,12 @@ type TitleProps = {
 
 const Title = ({ group: { name, id }, showEdit }: TitleProps) => (
   <NavigationTab title={name}>
+    <NavigationLink to={`/interestgroups/`}>
```

Comment:
You only need these ```{`...`} ``` if you use variables in there. With simple strings like this, just write `to="/interestgroups/"`.